modules/migration-debugging-velero-resources.adoc

```diff
@@ -61,6 +61,25 @@ $ oc -n {namespace} exec deployment/velero -c velero -- ./velero \
   backup describe 0e44ae00-5dc3-11eb-9ca8-df7e5254778b-2d8ql
 ----
 
+The following types of restore errors and warnings are shown in the output of a `velero describe` request:
+
+* `Velero`: A slice of messages related to the operation of Velero itself (for example, messages related to connecting to the cloud, reading a backup file, etc.)
+* `Cluster`: A slice of messages related to backup or restore of cluster-scoped resources
+* `Namespaces`: A map of namespace name to slice of messages related to backup or restore namespace-scoped resources
```

Comment:
> Anyway...I'll change "slice" to "list" ...

It still says slice here.

https://redhat-internal.slack.com/archives/C0144ECKUJ0/p1707406131362059?thread_ts=1707388097.633329&cid=C0144ECKUJ0

Reply:
```suggestion
* `Velero`: A list of messages related to the operation of Velero itself (for example, messages related to connecting to the cloud, reading a backup file, etc.)
* `Cluster`: A list of messages related to backup or restore of cluster-scoped resources
* `Namespaces`: A list of list of messages related to backup or restore namespace-scoped resources
```